proto/umee/leverage/v1/tx.proto

```diff
@@ -89,18 +89,18 @@ message MsgRepay {
 message MsgLiquidate {
   // Liquidator is the account address performing a liquidation and the signer
   // of the message.
-  string                   liquidator   = 1;
+  string liquidator = 1;
```

Comment:
Note: This file's diff is the result of running `make proto-all` (specifically `proto-fmt` step)